### PR TITLE
Create simple Makefile

### PR DESCRIPTION
**Pull request by aviau**
Based on https://github.com/Flared/purple-icq/pull/3

### CODE REVIEW

**Comment by aviau** [comment]
@isra17 review plz! (based on https://github.com/Flared/purple-icq/pull/3)

File: Makefile [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+	cp target/debug/libpurple_icq.so ~/.purple/plugins/libpurple_icq.so
+
+.PHONY: uninstall-user
+uninstall-user:
+	rm -f ~/.purple/plugins/libpurple_icq.so
+
+.PHONY:
+install:
+	mkdir -p /usr/lib/purple-2/
+	cp target/debug/libpurple_icq.so /usr/lib/purple-2/libpurple_icq.so
```
Comment (leameow) [inline Makefile:28]: `~/.purple/plugins/libpurple_icq.so`? Works on my machine and doesn't require root.

File: Makefile [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+install-user: build
+	mkdir -p ~/.purple/plugins
+	cp target/debug/libpurple_icq.so ~/.purple/plugins/libpurple_icq.so
+
+.PHONY: uninstall-user
+uninstall-user:
+	rm -f ~/.purple/plugins/libpurple_icq.so
+
+.PHONY:
+install:
```
Comment (leameow) [inline Makefile]: shouldn't it depends on `target/debug/libpurple_icq.so` and `target/debug/libpurple_icq.so` depends on `build`?

File: Makefile (hunk 1 above)
Comment (aviau) [reply Makefile:28]: The other target does this already (install-user).

File: Makefile (hunk 2 above)
Comment (aviau) [reply Makefile]: Ahh right!

I just added a dependency on ``build``, because cargo is already smart enough to check timestamps.

**Comment by aviau** [comment]
I immagine that resolving the  conversations means lgtm :3

**Pull request merged**